src/components/custom-field-input-single-choice/custom-field-input-single-choice.jsx

```diff
@@ -104,7 +104,7 @@ const CustomFieldInputSingleChoice = forwardRef(function CustomFieldInputSingleC
         readOnly={props.readOnly}
         required={props.required}
         tooltip={props.tooltip}
-        value={value[0] !== -1 ? choices.find(choice => choice.id === value[0]) : undefined}
+        value={value[0] !== -1 ? (choices.find(choice => choice.id === value[0]) || null) : null}
```

Comment:
This is probably something we want to come back and look for later. We were (I think unintentionally) hitting cases where `choices.find...` was coming back `undefined`.